### PR TITLE
GUI fix camera

### PR DESCRIPTION
**Pull request by Sinan-Karakaya**
Fix the camera centering of the app at the start of the game.

Written by @Nioroe

### CODE REVIEW

**Review by noerio — APPROVED** [review]
lack of documentation, seems good

File: zappy_gui/src/Map/Map.cpp [hunk 1]
```diff
@@ -68,12 +74,12 @@ void zp::Map::addRock(std::shared_ptr<zp::IEntity> rock)
     m_rocks.push_back(rock);
 }
 
-std::vector<std::shared_ptr<zp::IEntity>> &zp::Map::getAliens()
+std::vector<std::shared_ptr<zp::IEntity>> zp::Map::getAliens() const
```
Comment (Sinan-Karakaya) [inline zappy_gui/src/Map/Map.cpp]: Passing by copy instead of reference, not good.

File: zappy_gui/src/Map/Map.cpp [hunk 2]
```diff
@@ -68,12 +74,12 @@ void zp::Map::addRock(std::shared_ptr<zp::IEntity> rock)
...
 }
 
-std::vector<std::shared_ptr<zp::IEntity>> &zp::Map::getAliens()
+std::vector<std::shared_ptr<zp::IEntity>> zp::Map::getAliens() const
 {
     return m_aliens;
 }
 
-std::vector<std::shared_ptr<zp::IEntity>> &zp::Map::getRocks()
+std::vector<std::shared_ptr<zp::IEntity>> zp::Map::getRocks() const
```
Comment (Sinan-Karakaya) [inline zappy_gui/src/Map/Map.cpp]: Same here

File: zappy_gui/src/Map/Map.cpp [hunk 3]
```diff
@@ -88,7 +94,7 @@ void zp::Map::removeAlien(int id)
     }
 }
 
-const std::vector<std::string> &zp::Map::getTeams() const
+const std::vector<std::string> zp::Map::getTeams() const
```
Comment (Sinan-Karakaya) [inline zappy_gui/src/Map/Map.cpp]: Again.

**Review by Sinan-Karakaya — COMMENTED** [review]
DO NOT PASS VECTORS BY COPY.

To fix ASAP @Nioroe

**Pull request merged**